BaseApp/COD146027.TXT

```diff
@@ -0,0 +1,156 @@
+OBJECT Codeunit 146027 DotNet_Certificate Test
+{
+  OBJECT-PROPERTIES
+  {
+    Date=08.07.18;
+    Time=21:49:13;
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    Subtype=Test;
+    OnRun=BEGIN
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      Assert@1000 : Codeunit 130000;
+
+    [Test]
+    [LineStart(75)]
```

Comment:
What are these LineStart tags? :-D

Reply:
They are for integration with VsCode, so F12 can navigate to the method source in the base-app. I think we only add them when exporting to new syntax now.